images/ubuntu.yaml

```diff
@@ -807,5 +807,11 @@ actions:
   types:
   - vm
 
+- trigger: post-files
+  action: |-
+    #!/bin/sh
+    set -eux
+    [ -f /etc/netplan/10-lxc.yaml ] && chmod 0600 /etc/netplan/10-lxc.yaml
```

Comment:
Why don't we set the correct perms when its creates rather than later though?

Reply:
Hi, I'm having the same issue with netplan and used your code to set permissions. I'm still getting "Cannot call openvswitch: ovsdb-server.service is not running" . Am I doing something wrong here? are there any further steps needed?

Reply:
@lukasz0617 can you confirm that `10-lxc.yaml` has the correct permission set in the container?

Reply:
First time creating new instance for a few weeks and I am NOW also seeing

> "Permissions for /etc/netplan/90-lxc.yaml are too open. Netplan configuration should NOT be accessible by others. WARNING:root:Cannot call Open vSwitch: ovsdb-server.service is not running"

with image 22.04 (unchanged apart from any automatic updating). Note that 90-lxc.yaml was a pushed file and the permissions were indeed too open - I simply `chmod 0600 /etc/netplan/90-lxc.yaml` - and then `netplan try` accepted the config without issue but still reported 

> "Cannot call openvswitch: ovsdb-server.service is not running"

The LXD set up has been in place for a while. Doesn't seem to create any issues, but clearly not right?